### DISCUSSION TITLE
V136

### DISCUSSION THREAD

**Discussion by Rem0o** [Rem0o/FanControl.Releases#1250] [Announcements]
* Sexy graph UI
* Failed initialization will now prompt the sensor settings
* Use -c or --config [CONFIG FILE] argument to change config at runtime
* Create shortcuts, bind them to hotkeys with your favorite app, or automate as you please.
* Various bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/Rem0o/FanControl.Releases/releases/tag/V136'>V136</a>.</em>

**Comment by askiiart** [comment]
🎶 She thinks my graph is sexy 🎶

**Reply by Rem0o** [reply]
C'mon, would you look at these curves?
![image](https://user-images.githubusercontent.com/14118956/200078150-14292b3d-b1e5-42c5-af7d-f9bd790deb82.png)

Hot damn!
I asked what these curves are for. Answer ? OnlyFans.
